### PR TITLE
Fix mingw-cross build

### PR DESCRIPTION
**Pull request by kinkie**
compat/os/mswindows.h:640: ::inet_pton has not been declared

### CODE REVIEW

**Review by yadij — APPROVED** [review]
LGTM

File: compat/os/mswindows.h [hunk 1]
```diff
@@ -639,6 +642,7 @@ inet_pton(int af, const void *src, char *dst)
 #endif
```
Comment (rousskov) [inline compat/os/mswindows.h:642]: You have added (a very helpful) `HAVE_DECL_INET_NTOP` comment to the `else` clause in inet_ntop(). I cannot attach a change request to the corresponding inet_pton() function, but it should have a similar comment (mentioning `HAVE_DECL_INET_PTON`).

Comment (rousskov) [reply compat/os/mswindows.h:642]: Addressed in d6716bc0b4a3045cf2997fe540a85454b0c6a9b2 under the EasierDone agreement.

**Review by rousskov — APPROVED** [review]
I adjusted PR title to emphasize that this is a build fix and fixed PR description formatting (by removing non-essential details). The latter change should clear the `M-failed-description` label.

I believe these small changes and the trivial fixup do not deserve another review round. I did _not_ clear the `M-cleared-for-merge` label set by Amos.

**Pull request closed without merging**